Dynatrace/dynatrace-operator
# Instrument the CSI Server with OTel Traces

## Description

Adds OTel instrumentation to the csidriver containers and links them to the webhook created span. 
In addition, the singletons were replaced with init functions.

[Jira](https://dt-rnd.atlassian.net/browse/K8S-9452)

## How can this be tested?

- Apply the otel secret as described in `doc/otel.md`.
- Apply a dynakube with cloud native and create some pods. 
- Check the otel traces in your tenant.
